### DISCUSSION TITLE
Erratic power results on channel b (switching from positive to negative values and back)

### DISCUSSION THREAD

**Discussion by Steamerzone** [emporia-vue-local/esphome#171] [Q&A] [answered]
Hello,

I've just installed the VUE2 in a 3L+N system with solar return, and I'm getting erratic results on channel b.

The power values on that channel switches from positive to negative about every second, the absolute value stays more or less the same.

![image](https://github.com/emporia-vue-local/esphome/assets/22254368/30f93e74-9ada-4815-b4c3-9a8f1ba6de33)
![image](https://github.com/emporia-vue-local/esphome/assets/22254368/481013f8-588d-4ceb-940b-974aa7355728)

I'm pretty sure i've connected everything on the right phase, could this be a firmware or hardware issue?

Any help is more than welcome.

Greetings,
Emile

**Comment by flaviut** [comment]
Is the correct color selected for this phase? Can you try the other colors?

For additional detail, you can try removing the `filters:` lines & looking at the graphs.

I think this is unlikely to be a hardware or a firmware issue.

**Comment by Steamerzone** [comment, marked as answer]
I've recieved the unit with 3x 50A CT clamps for the main phases, they supplied three 2.5mm to 3.5 adapters for connecting to the VUE 2, one of the adapters was bad. (open circuit)

Solved!. Thanks.
